docs/upgrade-guides/node-to-express.mdx

```diff
@@ -70,7 +70,7 @@ app.listen(port, () => {
 
 ## `requireAuth()` usage changes
 
-In the Node SDK, `requireAuth()` wrapped route handlers. With the Express SDK, you must instead pass `requireAuth` as a middleware function before your handler instead of wrapping your handler. Additionally, `requireAuth` will now redirect un-authenticated users to the sign in page, where previously it would emit an error.
+In the Node SDK, `requireAuth()` wrapped route handlers. With the Express SDK, you must instead pass `requireAuth)` as a middleware function before your handler instead of wrapping your handler. Additionally, `requireAuth)` will now redirect un-authenticated users to the sign in page, where previously it would emit an error.
```

Comment:
```suggestion
In the Node SDK, `requireAuth()` wrapped route handlers. With the Express SDK, you must instead pass `requireAuth)` as a middleware function before your handler instead of wrapping your handler. Additionally, `requireAuth()` will now redirect un-authenticated users to the sign in page, where previously it would emit an error.
```